src/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wbpack Exercise</title>
+    <script
+      src="https://kit.fontawesome.com/4339e938ba.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="clear">
+          <i class="fa-solid fa-arrows-rotate"></i>
+        </div>
+      </header>
+      <section class="add-to-do">
+        <h1>TO DO List</h1>
+        <input type="text" id="input" placeholder="Add to-do" />
+        <button class="add">Add</button>
+        <section class="content">
+          <ul id="list">
+            <!-- <li class="item">
+              <div>
+              <i class="fa-regular fa-square"></i>
+
+              <p class="text">Drink Coffe</p>
+              </div>
+              <i class="fa-solid fa-trash"></i>
+            </li> -->
```

Comment:
- Please remove all commented code from your file. This would help make your code look more readable and easier to maintain 👍 👍